go.mod

```diff
@@ -6,11 +6,13 @@ require (
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/coredns/corefile-migration v1.0.11
+	github.com/coreos/ignition v0.35.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/docker/distribution v2.7.1+incompatible
 	github.com/drone/envsubst v1.0.3-0.20200709223903-efdb65b94e5a
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/fatih/color v1.7.0
+	github.com/flatcar-linux/container-linux-config-transpiler v0.6.2-0.20201012083443-7abced2731c3
```

Comment:
Could we use a tag that doesn't rely on commits?

Reply:
We need https://github.com/kinvolk/container-linux-config-transpiler/pull/9/commits/75d4b209a329e1ab7399127a2469af795249e663 to be able to import it, which is currently not tagged. Let me see what I can do about that.

Reply:
Let's try to before we merge this

Reply:
Could we get this fixed to a tagged release before merging?

Reply:
This has been tagged today at https://github.com/kinvolk/container-linux-config-transpiler/releases/tag/v0.9.1. I'll update to that when I find time to rebase this PR.

Reply:
Fixed.